apps/cow-tools/src/app/milkman/utils/type.ts

```diff
@@ -0,0 +1,9 @@
+export enum TransactionStatus {
+  TO_BE_EXECUTED = "To be executed",
+  MILKMAN_CREATED = "Milkman created",
+  ORDER_PLACED = "Order placed",
+  EXECUTING = "Executing",
+  EXECUTED = "Executed",
+  CANCELATION_TO_BE_EXECUTED = "Cancelation to be executed",
+  CANCELED = "Canceled",
+}
```

Comment:
Will leave as a reminder, but I'd redo the whole tx logic. We can get inspo from Milkman's/Balancer/UNI UIs but can take a read at the code as well as they're all open source